src/lib/discord/commands.ts

```diff
@@ -26,22 +27,21 @@ function createCommandsMap(commands: any[]) {
       ) => {
         const somethingWentWrongResponse = '🤕 Something went wrong'
         try {
+          await interaction.deferReply({ephemeral: true})
```

Comment:
does this have any substantial impact to the other commands? is it possible to handle the deferred reply in the admin command handler?

Reply:
I didn't see any change with the other commands but I can look into that